build/stf-run-ci/tasks/create_catalog.yml

```diff
@@ -30,7 +30,14 @@
   ansible.builtin.set_fact:
     internal_registry: "{{ builder_dockercfg_auth_results['image-registry.openshift-image-registry.svc:5000'] | to_json }}"
 
-- when: query('kubernetes.core.k8s', api_version='v1', kind='Secret', resource_name='service-telemetry-framework-index-dockercfg', namespace=namespace) | length == 0
+- name: "Get Secrets to check for service-telemetry-framework-index-dockercfg"
+  ansible.builtin.command:
+    cmd: oc get secret -n {{ namespace }} service-telemetry-framework-index-dockercfg
```

Comment:
Why are you preferring cmd to k8s for these?

Reply:
This change is not yet testable in the CI we have. It is lower priority than the others.

Reply:
Makes things easier to replicate, too. I think that's a good tradeoff (vs. the speed and encapsulation) for CI purposes